docs/microservices.json

```diff
@@ -52,6 +52,44 @@
                 }
             ]
         },
+        {
+            "id": "letters-producer",
```

Comment:
We would also need to represent `service bus` I guess ?

Reply:
I think this is just for applications that we've built